modulesync.gemspec

```
@@ -29,5 +29,5 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'gitlab', '>=4', '<6'
   spec.add_runtime_dependency 'octokit', '>=4', '<10'
   spec.add_runtime_dependency 'puppet-blacksmith', '>= 3.0', '< 9'
-  spec.add_runtime_dependency 'thor', '1.3.0'
+  spec.add_runtime_dependency 'thor', '1.3.2'
```

It was pinned in 7fe82ae1768d648f2ee33beff8331ce88b316114. If only 1.3.1 was broken then I'd prefer this:
```suggestion
  spec.add_runtime_dependency 'thor', '~> 1.3', '!= 1.3.1'
```